src/chk/check.pb-c.c

```diff
@@ -0,0 +1,281 @@
+{
+  { "CF_AUTO", 4 },
+  { "CF_DRYRUN", 1 },
+  { "CF_FAILOUT", 3 },
+  { "CF_NONE", 0 },
+  { "CF_RESET", 2 },
+};
+const ProtobufCEnumDescriptor chk__check_flag__descriptor =
+{
+  PROTOBUF_C__ENUM_DESCRIPTOR_MAGIC,
+  "chk.CheckFlag",
+  "CheckFlag",
+  "Chk__CheckFlag",
+  "chk",
+  5,
+  chk__check_flag__enum_values_by_number,
+  5,
+  chk__check_flag__enum_values_by_name,
+  3,
+  chk__check_flag__value_ranges,
+  NULL,NULL,NULL,NULL   /* reserved[1234] */
+};
+static const ProtobufCEnumValue chk__check_inst_status__enum_values_by_number[7] =
+{
+  { "CIS_INIT", "CHK__CHECK_INST_STATUS__CIS_INIT", 0 },
+  { "CIS_RUNNING", "CHK__CHECK_INST_STATUS__CIS_RUNNING", 1 },
+  { "CIS_COMPLETED", "CHK__CHECK_INST_STATUS__CIS_COMPLETED", 2 },
+  { "CIS_STOPPED", "CHK__CHECK_INST_STATUS__CIS_STOPPED", 3 },
+  { "CIS_FAILED", "CHK__CHECK_INST_STATUS__CIS_FAILED", 4 },
+  { "CIS_PAUSED", "CHK__CHECK_INST_STATUS__CIS_PAUSED", 5 },
+  { "CIS_CRASHED", "CHK__CHECK_INST_STATUS__CIS_CRASHED", 6 },
+};
+static const ProtobufCIntRange chk__check_inst_status__value_ranges[] = {
+{0, 0},{0, 7}
+};
+static const ProtobufCEnumValueIndex chk__check_inst_status__enum_values_by_name[7] =
+{
+  { "CIS_COMPLETED", 2 },
+  { "CIS_CRASHED", 6 },
+  { "CIS_FAILED", 4 },
+  { "CIS_INIT", 0 },
+  { "CIS_PAUSED", 5 },
+  { "CIS_RUNNING", 1 },
+  { "CIS_STOPPED", 3 },
+};
+const ProtobufCEnumDescriptor chk__check_inst_status__descriptor =
+{
+  PROTOBUF_C__ENUM_DESCRIPTOR_MAGIC,
+  "chk.CheckInstStatus",
+  "CheckInstStatus",
+  "Chk__CheckInstStatus",
+  "chk",
+  7,
+  chk__check_inst_status__enum_values_by_number,
+  7,
+  chk__check_inst_status__enum_values_by_name,
+  1,
+  chk__check_inst_status__value_ranges,
+  NULL,NULL,NULL,NULL   /* reserved[1234] */
+};
+static const ProtobufCEnumValue chk__check_pool_status__enum_values_by_number[6] =
+{
+  { "CPS_UNCHECKED", "CHK__CHECK_POOL_STATUS__CPS_UNCHECKED", 0 },
+  { "CPS_CHECKING", "CHK__CHECK_POOL_STATUS__CPS_CHECKING", 1 },
+  { "CPS_CHECKED", "CHK__CHECK_POOL_STATUS__CPS_CHECKED", 2 },
+  { "CPS_FAILED", "CHK__CHECK_POOL_STATUS__CPS_FAILED", 3 },
+  { "CPS_PAUSED", "CHK__CHECK_POOL_STATUS__CPS_PAUSED", 4 },
+  { "CPS_PENDING", "CHK__CHECK_POOL_STATUS__CPS_PENDING", 5 },
+};
+static const ProtobufCIntRange chk__check_pool_status__value_ranges[] = {
+{0, 0},{0, 6}
+};
+static const ProtobufCEnumValueIndex chk__check_pool_status__enum_values_by_name[6] =
+{
+  { "CPS_CHECKED", 2 },
+  { "CPS_CHECKING", 1 },
+  { "CPS_FAILED", 3 },
+  { "CPS_PAUSED", 4 },
+  { "CPS_PENDING", 5 },
+  { "CPS_UNCHECKED", 0 },
+};
+const ProtobufCEnumDescriptor chk__check_pool_status__descriptor =
+{
+  PROTOBUF_C__ENUM_DESCRIPTOR_MAGIC,
+  "chk.CheckPoolStatus",
+  "CheckPoolStatus",
+  "Chk__CheckPoolStatus",
+  "chk",
+  6,
+  chk__check_pool_status__enum_values_by_number,
+  6,
+  chk__check_pool_status__enum_values_by_name,
+  1,
+  chk__check_pool_status__value_ranges,
+  NULL,NULL,NULL,NULL   /* reserved[1234] */
```

Comment:
(style)  space required after that ',' (ctx:VxV)
(style)  space required after that ',' (ctx:VxV)
(style)  space required after that ',' (ctx:VxV)